Update imu-calibration.md (#274)
SlimeVR/SlimeVR-Docs-Site: src/server/imu-calibration.md

Optionally, you can perform temperature calibration for BMI160 IMUs, which is a more advanced process but significantly reduces drift:

> -With a 3d printer

Reviewer: ```suggestion
#### With a 3d printer
```

1. Place your trackers in a fridge or freezer for a period of time to cool them down to below 15°C. If you're unsure about the temperature of your trackers, you can check it in the SlimeVR Server when the tracker is turned on.
1. Turn on your trackers and gradually heat them, calibration will automatically begin if the temperature is below 15°C.
   - You can use something like a 3D printer bed or a heat gun, but be careful not to overheat your battery or melt your case.
   - It is important to not rush this. Temperature calibration should take at least 15 minutes, otherwise you risk having a partial calibration, which can lead to increased drift.
1. The calibration will be complete and automatically saved once the IMU reaches 45°C.

> -With a hairdryer

Reviewer: ```suggestion
#### With a hairdryer
```

> Step 1. Place your tracker in a fridge or freezer for a short period of time(about 10 minutes) to cool it down to below 15°C. 
> Step 2. Get the tracker out and put it on a stable underground. Turn on the Slimeserver/more info to see all details about your slime and it's temperature. Turn it on and let it sit until it reaches operating temperatures naturally; Don't use a hairdryer yet! When you put your tracker closer to your computer you can reach a higher temperature quicker.
> Step 3. Wait until your tracker's temperature levels out. Warm it up with your hairdryer to around 43°C, do not reach 45°C yet.
> Step 4. Let it cool down naturally to operating temperature again.
> Step 5. 30 degrees? Airdry it to 46 degrees to squeeze out the last points for your TCAL and let it cool down.
> Step 6. Check if you did well with this USB webui checking tool:  https://ilyasnow.github.io/TCAL-Checker/
> 

Reviewer: ```suggestion
1. Place your tracker in a fridge or freezer for a short period of time(about 10 minutes) to cool it down to below 15°C. 
1. Get the tracker out and put it on a stable underground.
   - Open the SlimeVR app -> Turn on developer mode -> In home at the bottom-right toggle more info, to see all details about your slime and it's temperature.
   - Turn it on and let it sit until it reaches operating temperatures naturally; Don't use a hairdryer yet! When you put your tracker closer to your computer you can reach a higher temperature quicker.
1. Wait until your tracker's temperature levels out. Warm it up with your hairdryer to around 43°C, do not reach 45°C yet.
1. Let it cool down naturally to operating temperature again.
1. 30 degrees? Airdry it to 46 degrees to squeeze out the last points for your TCAL and let it cool down.
1. Check if you did well with the [USB WebUI checking tool](https://ilyasnow.github.io/TCAL-Checker/)

```

Optionally, you can perform temperature calibration for BMI160 IMUs, which is a more advanced process but significantly reduces drift:

> #### With a 3d printer

Reviewer 2: ```suggestion
#### With a 3d printer

```

1. Place your trackers in a fridge or freezer for a period of time to cool them down to below 15°C. If you're unsure about the temperature of your trackers, you can check it in the SlimeVR Server when the tracker is turned on.
1. Turn on your trackers and gradually heat them, calibration will automatically begin if the temperature is below 15°C.
   - You can use something like a 3D printer bed or a heat gun, but be careful not to overheat your battery or melt your case.
   - It is important to not rush this. Temperature calibration should take at least 15 minutes, otherwise you risk having a partial calibration, which can lead to increased drift.
1. The calibration will be complete and automatically saved once the IMU reaches 45°C.

> #### With a hairdryer

Reviewer 2: ```suggestion
#### With a hairdryer

```

> 1. Place your tracker in a fridge or freezer for a short period of time(about 10 minutes) to cool it down to below 15°C. 

Reviewer 2: ```suggestion
1. Place your tracker in a fridge or freezer for a short period of time (about 10 minutes) to cool it down to below 15°C. 
```

> 1. Get the tracker out and put it on a stable underground.
>    - Open the SlimeVR app -> Turn on developer mode -> In home at the bottom-right toggle more info, to see all details about your slime and it's temperature.

Reviewer 2: ```suggestion
1. Take the tracker out and put it on a stable surface like a table.
   - Open the SlimeVR app -> Turn on developer mode -> One the home tab at the bottom-right, toggle "More info" to see all details about your Slime and it's temperature.
```

Reviewer 3: I would rename home tab to home page

   - Turn it on and let it sit until it reaches operating temperatures naturally; Don't use a hairdryer yet! When you put your tracker closer to your computer you can reach a higher temperature quicker.
1. Wait until your tracker's temperature levels out. Warm it up with your hairdryer to around 43°C, do not reach 45°C yet.
1. Let it cool down naturally to operating temperature again.

> 1. 30 degrees? Airdry it to 46 degrees to squeeze out the last points for your TCAL and let it cool down.

Reviewer 2: ```suggestion
1. Once around 30°C, use the hairdryer to get it to around 46°C to finish measuring the last points for your TCAL, then let it cool down.
```
